### PR TITLE
Adds support for position precision settings

### PR DESCRIPTION
**Pull request by jp-bennett**

### CODE REVIEW

File: src/mesh/NodeDB.cpp [hunk 1]
```diff
@@ -195,6 +195,13 @@ void NodeDB::installDefaultConfig()
     config.position.position_broadcast_smart_enabled = true;
     config.position.broadcast_smart_minimum_distance = 100;
     config.position.broadcast_smart_minimum_interval_secs = 30;
+
+    config.position.channel_precision_count = 8;
+    config.position.channel_precision[0] = 32;
```
Comment (thebentern) [inline src/mesh/NodeDB.cpp:200]: Technically, we shipped the protobufs themselves on the last release. Should we create a migration of sorts to init this for the first time?

Comment (jp-bennett) [reply src/mesh/NodeDB.cpp:200]: Ironically we've come to a bit different conclusion on how this should work. Will be dropping some of this work in favor of that approach.

**Comment by caveman99** [comment]
@jp-bennett Can this be closed?

File: src/mesh/NodeDB.cpp (hunk 1 above)
Comment (aaaidan) [reply src/mesh/NodeDB.cpp:200]: > Will be dropping some of this work in favor of that approach.

@jp-bennett can you say more about this? Has a precision masking feature already been implemented or rolled out?

Comment (jp-bennett) [reply src/mesh/NodeDB.cpp:200]: Yes, it's been in firmware for quite a while. We set the precision per channel.

Comment (aaaidan) [reply src/mesh/NodeDB.cpp:200]: Oh, that’s fantastic. Is this setting in all apps? I can’t find this in iOS, for example, but don’t know where to look.

I would love to share my general location I would just like to make sure my exact position is never transmitted without my intent.

**Pull request closed without merging**